Resources/Prototypes/Entities/Objects/Devices/pda.yml

```
@@ -284,6 +284,9 @@
     state: pda-clown
   - type: Slippery # secretly made of bananium
   - type: StepTrigger
+    triggerGroups: # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)